zproject/prod_settings_template.py

```diff
@@ -152,6 +152,23 @@
     # "zproject.backends.GenericOpenIdConnectBackend",  # Generic OIDC integration, setup below
 )
 
+## JWT-based users login and API keys fetching.
+##
+## If you want to login users or fetch API keys via JWT in Zulip,
+## you need to set the secret key and algorithm to use to validate
+## JWT tokens received.
+# JWT_AUTH_KEYS: Dict[str, Any] = {
+#     # Subdomain for which you want to enable the service.
+#     "zulip": {
+#         # Shared secret key used to validate jwt tokens. This key
+#         # needs to be securely, randomly generated.
+#         "key": "xxxxxx",
+#         # Algorithm with which the jwt token are signed.
+#         "algorithms": ["HS256"],
+#     }
+# }
```

Comment:
I amended the original commit to comment out these lines, because otherwise this config was going to be enabled by default for all new installations

Reply:
makes sense